docs/docs/domain.mdx

```diff
@@ -463,66 +468,177 @@ stories:
+- `intent`: Only applies the mapping when this intent is predicted.
+- `not_intent`: Does not apply the mapping when this intent is predicted
 
-:::
+Note that if you choose not to define the parameter `intent`, the slot mapping will apply regardless of the intent
+of the message as long as the intent is not listed under the `not_intent` parameter.
+
+```yaml-rasa
+slots:
+  slot_name:
+    type: any
+    mappings:
+    - type: from_intent
+      value: my_value
+      intent: intent_name
+      not_intent: excluded_intent
+```
+
+#### from_trigger_intent
+
+The `from_trigger_intent` mapping will fill slot `slot_name` with value `my_value`
+if a form is activated by a user message with intent `intent_name`.
+The slot mapping will not apply if the intent of the message is
+`excluded_intent`.
+
+A slot with `from_trigger_intent` mapping must always be listed in the form's `required_slots`.
+You can verify if your domain is compliant with this requirement by running
+CLI command [`rasa data validate`](./command-line-interface.mdx#rasa-data-validate).
+
+```yaml-rasa
+slots:
+  slot_name:
+    type: any
+    mappings:
+    - type: from_trigger_intent
+      value: my_value
+      intent: intent_name
+      not_intent: excluded_intent
+      conditions:
+      - active_loop: your_form
+forms:
+  your_form:
+    required_slots:
+    - slot_name
+```
 
+### Mapping Conditions
 
+To apply a slot mapping only within the context of a form, specify
+the name of the form in the `conditions` key of a slot mapping. Conditions list the form name(s)
+for which the mapping is applicable in the `active_loop` key.
 
-To disable auto-filling for a particular slot, you can set the
-`auto_fill` attribute to `false` in the domain file:
+Conditions can also include the name of the `requested_slot`. If `requested_slot` is not mentioned,
+then the slot will be set if relevant information is extracted, regardless of which slot is being
+requested by the form.
 
 ```yaml-rasa
 slots:
-  name:
+  slot_name:
     type: text
     mappings:
-    - type: from_entity
-      entity: name
+    - type: from_text
+      intent: intent_name
+      conditions:
+      - active_loop: your_form
+        requested_slot: slot_name
+      - active_loop: another_form
+```
+
+:::note
+If `conditions` are not included in a slot mapping, the slot mapping will be applicable regardless of whether
+any form is active. As long as a slot is listed in a form's `required_slots`, the form will prompt for the slot
+if it is empty when the form is activated.
+:::
+
+### Custom Slot Mappings
+
+You can define custom slot mappings using [slot validation actions](./slot-validation-actions.mdx) when none of the
+predefined mappings fit your use case. You must define this slot mapping to be of type `custom`, for example:
+
+```yaml
+slots:
+  day_of_week:
+    type: text
+    mappings:
+    - type: custom
+      action: action_calculate_day_of_week
```

Comment:
```suggestion

You can define custom slot mappings using [slot validation actions](./slot-validation-actions.mdx) when none of the
predefined mappings fit your use case. You must define this slot mapping to be of type `custom`, for example:

```yaml
slots:
  day_of_week:
    type: text
    mappings:
    - type: custom
      action: action_calculate_day_of_week
\```

You can also use the `custom` slot mapping to list slots that will be filled by arbitrary custom actions in the course of a conversation, by listing the type and no specific. For example:

```yaml
slots:
  handoff_completed:
    type: boolean
    mappings:
    - type: custom
\```

This slot will not be updated on every user turn, but only once a custom action that returns a `SlotSet` event for it is predicted. 

```

Can't get the backticks to play along; Is this true though? To maintain the previous behaviour of a slot that is created without being filled on each turn user's will need to know this

Reply:
Yup, that's correct, I'll add that 👍🏼